src/ResourceManager/Compute.ManagedService/AzureRM.Compute.ManagedService.psd1

```diff
@@ -0,0 +1,129 @@
+#
+# Module manifest for module 'PSGet_AzureRM.Compute.ManagedService'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 01/19/2018
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = '.\ConvertTo-AzureRmVhd.psm1'
+
+# Version number of this module.
+ModuleVersion = '0.1.0'
+
+# Supported PSEditions
+# CompatiblePSEditions = @()
+
+# ID used to uniquely identify this module
+GUID = '5C5F634F-B323-4822-B8E4-695DD0110AF9'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = 'Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Microsoft Azure PowerShell - Managed Compute service cmdlets for Azure Resource Manager.'
+
+# Minimum version of the Windows PowerShell engine required by this module
+PowerShellVersion = '3.0'
+
+# Name of the Windows PowerShell host required by this module
+# PowerShellHostName = ''
+
+# Minimum version of the Windows PowerShell host required by this module
+# PowerShellHostVersion = ''
+
+# Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+DotNetFrameworkVersion = '4.5.2'
+
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+CLRVersion = '4.0'
+
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
+
+# Modules that must be imported into the global environment prior to importing this module
+RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '4.1.1'; })
```

Comment:
Please update to 4.2.0.

Also, please add a Changelog for this new directory

Reply:
OK